common/autoware_universe_utils/src/geometry/random_concave_polygon.cpp

```diff
@@ -0,0 +1,410 @@
+// Copyright 2024 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "autoware/universe_utils/geometry/random_concave_polygon.hpp"
+
+#include "autoware/universe_utils/geometry/boost_geometry.hpp"
+
+#include <boost/geometry/algorithms/convex_hull.hpp>
+#include <boost/geometry/algorithms/correct.hpp>
+#include <boost/geometry/algorithms/intersects.hpp>
+#include <boost/geometry/strategies/agnostic/hull_graham_andrew.hpp>
```

Comment:
Why did you include this? I am trying my best to remove all these: https://github.com/autowarefoundation/autoware.universe/pull/8528

Reply:
Seems like it was removed from boost: https://live.boost.org/doc/libs/1_86_0/libs/geometry/doc/html/geometry/release_notes.html#geometry.release_notes.boost_1_76

Reply:
Autoware is currently targeting ROS2 humble running on Ubuntu 22.04. The `boost` version is thus `1.74` where we still need to include this header.

Reply:
Ah, I was a bit hasty with my comment it seems. All other instances included it but didn't use it. Here the included head is actually used. We'll have to figure this out later then.